File: rudi-microservice/rudi-microservice-strukture/rudi-microservice-strukture-core/src/main/java/org/rudi/microservice/strukture/core/bean/criteria/AddressRoleSearchCriteria.java
/**
 * 
 */
package org.rudi.microservice.strukture.core.bean.criteria;

import org.rudi.microservice.strukture.core.bean.AddressType;

import lombok.Data;

/**
 * @author FNI18300
 *
 */
@Data
public class AddressRoleSearchCriteria {
	private String code;
	private AddressType type;
	private Boolean active;
}

File: rudi-application/rudi-application-front-office/angular-project/src/app/core/bean/strukture/address-role-search-criteria.ts
import {StampedSearchCriteria} from '@core/bean/stamped-search-criteria';
import {AddressType} from 'micro_service_modules/strukture/strukture-model';

export class AddressRoleSearchCriteria extends StampedSearchCriteria {
    code?: string;
    type?: AddressType;
}
